Sources/Player/PlaybackEngine/Internal/AVQueuePlayer/Legacy/AVQueuePlayerWrapperLegacy.swift

```diff
@@ -154,9 +159,26 @@ final class AVQueuePlayerWrapperLegacy: GenericMediaPlayer {
 
 			self.delegates.seeking(in: asset)
 
-			let completed = await self.player.seek(to: time)
-			guard completed, currentItem == self.player.currentItem, self.player.timeControlStatus == .playing else {
-				return
+			let seekTime = CMTime(seconds: time, preferredTimescale: 1000)
+			if self.shouldPauseAndPlayAroundSeek {
+				if self.player.timeControlStatus == .playing {
+					self.isSeeking = true
+					await self.player.pause()
+				}
+
+				let completed = await self.player.seek(to: time)
+				self.isSeeking = false
+				await self.player.play()
+
+				guard completed, currentItem == self.player.currentItem else {
+					return
+				}
+			} else {
+				self.player.seek(to: seekTime) { completed in
+					guard completed, currentItem == self.player.currentItem, self.player.timeControlStatus == .playing else {
+						return
+					}
+				}
```

Comment:
Did you forget to return to the `await seek` version here?

Reply:
oh yes, I changed back after checking that and didn't commit, it is in staging, thanks for spotting!